gm4_liquid_tanks/data/gm4_liquid_tanks/functions/main.mcfunction

```diff
@@ -5,7 +5,8 @@ execute as @e[type=item,nbt={Item:{id:"minecraft:glass",Count:1b},OnGround:1b}]
 execute as @e[type=armor_stand,tag=gm4_liquid_tank] at @s run function gm4_liquid_tanks:process
 
 #tank display maintenance
-execute as @e[type=armor_stand,tag=gm4_liquid_tank_display] run data merge entity @s {Fire:20000}
+# 1.18 NOTE; This 16->17 backward compatibility may be removed.
+execute as @e[type=armor_stand,tag=gm4_liquid_tank_display] run data merge entity @s {HasVisualFire:1}
```

Comment:
Why not move this to a slower clock?

Reply:
This module does not already have any slower clocks running, and creating a whole one didn't really seem worth the effort when the 1.16 version was already doing a nbt write every 1 ticks anyway.